.travis.yml

```diff
@@ -52,6 +52,7 @@ jobs:
         - tox -e py37
         - echo 'travis_fold:end:cocotb_py37'
       after_success: &travis_linux_post
+        - find . -name ".library_coverage" | xargs coverage combine .coverage
```

Comment:
This overwrites the output of the pytest run. pytest should deposit it's results to `.library_coverage`.